src/js/relayclient/ServerHelper.js

```diff
@@ -143,7 +138,7 @@ class ServerHelper {
   defaultCalculateRelayScore (relay) {
     // basic score is trasnaction fee (which is %)
     // higher the better.
-    let score = 1000 - relay.transactionFee
+    let score = 1000 - relay.pctRelayFee
```

Comment:
this default score is broken: 
I can put `baseFee = 1e18, pctRelayFee=0`, and I'll always be ordered as first...
The real calculation should estimate the (inner) tx gas price, and use the `base+estimate*pct` for score.

Reply:
also, misisng tests for various baseFee values

Reply:
Can I pull this task to a different PR? This one is already getting bloated and will get in the way of merging with Shahaf

Reply:
what you're saying is "its a baseFee pull request, but a different PR will actually make it use the baseFee"

Reply:
then rename it. its "initial support", and in the comment mention that its incomplete.

Reply:
Done